libqtile/group.py

```diff
@@ -460,12 +540,12 @@ def cmd_prev_window(self):
             return
         if self.current_window.floating:
             nxt = self.floating_layout.focus_previous(self.current_window) or \
-                self.layout.focus_last() or \
-                self.floating_layout.focus_last(group=self)
+                  self.layout.focus_last() or \
```

Comment:
Can you drop all the reformatting bits?

Reply:
Sorry for those, that seems to be something Pycharm does automatically. I'll get rid of those when cleaning up the formatting.